CONTRIBUTING.md

```diff
@@ -1,167 +1,113 @@
+This method of adding a new library using a single `package.json` file is recommended for beginners. You can do this directly from GitHub and don't need to use the command line. Before following the steps below, remember to follow our [conventions for branches, commits and pull requests](#).
 
-* The new library must have at least one officially public accessable repository and open source license.
+* [Fork the CDNJS repo](https://github.com/cdnjs/cdnjs/fork) to your GitHub account.
+* [Create a new branch](https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/).
+* Navigate to the `ajax/libs` directory.
+* Click on "Create new file".
+* For the file name, enter `LIBRARY_NAME/package.json`. LIBRARY_NAME will be the name of the directory in `ajax/libs`. For example, `jquery/package.json`.
+* Compose a valid `package.json`file following the conventions listed above. Use minified versions for `filename` in `package.json` even if they don't exist. Our maintainers will minify the library when hosting it. For example, `jquery.min.js`.
+* Make a new commit and submit the pull request.
 
-## B. pull request
+#### Pre-Flight Checks
 
-### a. Sparse checkout and shallow clone
-* [The short document of how to use sparse checkout and shallow clone is here](https://github.com/cdnjs/cdnjs/blob/master/documents/sparseCheckout.md)
-* Sparse checkout and shallow clone can totally raise efficient of working with cdnjs repo and avoid the problem of case-insensitive filesystem, for detail reason, see it below.
-* The minor reason that you should use sparse checkout and shallow clone:
-  * There is no need to clone the whole history of cdnjs repo if you are not tracing issues or maintaining this project, and there is no need to checkout all the libs on cdnjs for the same reason(unless you want to access the libs without the Internet).
-* And the major reason:
-  * The size of the git repo behind cdnjs is about 2.7GB, but the extracted files will use about 70GB space on your filesystem. If you are using case-insensitive filesystem(usually on Windows and MacOS), you may have problem with [**S**ortable](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/Sortable) and [**s**ortable](https://github.com/cdnjs/cdnjs/tree/master/ajax/libs/sortable), we have an issue disscussion here: [#3650](https://github.com/cdnjs/cdnjs/issues/3650)
+* Have you complied with all our conventions?
+* Have you followed the library's directory structure?
+* Does a valid and accurate `package.json` file exist for the library?
+* Have you minified CSS and JavaScript assets?
+* Did all tests pass when you ran `npm test`?
 
+#### Miscellaneous Information
 
+* We recommend working on a Unix-like environment (for example, GNU/Linux or BSD distributions except macOS) due to how git works.
```

Comment:
"working on" grammatical change "working in "  
"except macOS" grammer==== "except for macOS"
Thanks